samanthas_telegram_bot/conversation/callbacks/registration/student.py

```diff
@@ -54,9 +54,17 @@ async def ask_communication_language_or_start_assessment_depending_on_learning_e
     query, data = await answer_callback_query_and_get_data(update)
 
     if data == "year_or_more":
```

Comment:
const?

Reply:
Nah, I'm not sure. These phrases have to match internal IDs of phrases in phrases.csv, so it's "magic" anyway. It depends on another file.

Reply:
Got it